D-lama-service/Data/ProjectEntities/DataPoint.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Data.ProjectEntities
+{
+    /// <summary>
+    /// Data point entity.
+    /// </summary>
+    public abstract class DataPoint : Entity
+    {
+        [Required]
+        public int DataPointIndex { get; set; }
```

Comment:
What for do we need the DataPointIndex?

Reply:
In order to indentify the index of the data points inside a project. All data points of all projects are in one table. For using the end points get range, delete range etc. we need these indices. The Id of a TextDataPoint is assigned globally over all projects, that is why I created the DataPointIndex attribute.

Reply:
Ah, okay, that makes sense. Thanks for explaining it to me!